src/core/src/op/strided_slice.cpp

```diff
@@ -325,11 +326,17 @@ bool op::v1::StridedSlice::constant_fold(OutputVector& output_values, const Outp
                 ? clone_with_new_inputs(OutputVector{inputs_values[0], begin, end, inputs_values[3]})->output(0)
                 : this->output(0);
 
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        if (const auto c = ov::get_constant_from_source(output)) {
-            OPENVINO_SUPPRESS_DEPRECATED_END
-            output_values[0] = c;
-            is_folded = true;
+        std::vector<Node*> nodes;
+        // Check if bounds can be evaluated and none of output nodes have disabled constant folding.
+        if (ov::could_propagate(output, nodes) && std::none_of(nodes.begin(), nodes.end(), [](const Node* n) {
+                return ov::pass::constant_folding_is_disabled(n);
+            })) {
+            OPENVINO_SUPPRESS_DEPRECATED_START
+            if (const auto c = ov::get_constant_from_source(output)) {
+                OPENVINO_SUPPRESS_DEPRECATED_END
+                output_values[0] = c;
```

Comment:
I guess that we need to copy runtime info here at least from StridedSlice node to this constant, but maybe from input nodes also.
Please refer `Node::constant_fold` implementation

Reply:
I follow the `Node` implementation which use all node's inputs to copy `rt_info`. In case of Strides copy from nodes which will be folded into constant.